### ISSUE TITLE
VO mobile moves from sim description to bottom toolbar 

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#304]
*Test device**
iPad. Air 2 
iPadOS 14.7.1 

**Problem description**
https://github.com/phetsims/qa/issues/702

Sometimes, when I first enter the sim using VO, the descriptions are read and then the VO moves to the preferences button. I have to then swipe left to be able to actually interact with the sim itself. 

**Steps to reproduce**
1. open the sim
2. turn on VO by pressing the home button 3 times
3. Keep swiping right to try and enter the play area. 

I have also seen VO enter the sim and go directly to the preferences button without reading descriptions at all. 

I am new to VO on the iPad. If this is expected, correct behavior please let me know. 

**Visuals**

https://user-images.githubusercontent.com/87318828/132382140-e54795e1-45b9-4fa5-8adf-e003e5231b0b.mov

**Comment by jessegreenberg** [comment]
Interesting, it could be related to #305 but I am not sure. If you tap on the Preferences button, I would expect the virtual cursor (black highlight) to move there, but the video is clearly swiping gestures. Perhaps the end of the gesture is too close to the preferences button and VO thinks focus should go there. Ill try to do gestures that look similar to what @Nancy-Salpepi was doing in this video and see if I am getting it to happen too.

**Comment by jessegreenberg** [comment]
I am quite sure this is the same as #305 now closing this and moving to that one since there are already more comments there.
